ui/app/mainui/SplashScreen.qml

```diff
@@ -5,25 +5,12 @@ import utils 1.0
 
 Item {
     id: root
-    anchors.fill: parent
-    visible: (opacity > 0.0001)
-    Behavior on opacity { NumberAnimation { duration: 250 }}
-
-    Timer {
-        running: true
-        interval: 2000
-        onTriggered: {
-            root.opacity = 0.0;
-        }
-    }
 
     AnimatedImage {
-        id: splashLogo
-        width: 127.88
-        height: 127.88
+        width: 128
+        height: 128
         anchors.centerIn: parent
-        source: Style.gif("status_splash_" + (Style.current.name))
-        playing: visible
+        source: Style.gif("status_splash_128_" + (Style.current.name))
```

Comment:
Isn't the `playing` attribute needed anymore?

Reply:
@richard-ramos ‘playing’ is defaulted to true for AnimatedImage, explicit set to true is not needed.

> Ideally refactors should be done incrementally instead of doing a big bang refactor like it was done here.

It’s a feature, not refactoring ;)

Reply:
Doing incremental changes is a concept that still applies to new features - It is easier to review a set of smaller PRs than a single one with 4,689 additions, 930 deletions / 113 files changed

Reply:
>  ‘playing’ is defaulted to true for AnimatedImage, explicit set to true is not needed.

I think the problem was that once we switch to the real app, the animation kept playing even if it was no longer visible. 

I guess the main problem is that we didn't remove the SplashScreen in that case, but still, we need to address the CPU issue and using `playing: visible` was a quick win.

Reply:
> Doing incremental changes is a concept that still applies to new features - It is easier to review a set of smaller PRs than a single one with 4,689 additions, 930 deletions / 113 files changed

@richard-ramos there are 2 commits (one is for keycard and one is for using splash screen as app loading animation screen), the second one was in a separate PR, here https://github.com/status-im/status-desktop/pull/6755 where the base was this branch and it was merged here. But anyhow... you're right, it could be in a few PRs, but from the other side all of them are too connected, but maybe I could do them by flow since the code covers 4 flows. The next big section is a settings part for the keycard, what would you like to see there, like separate commit for each flow or separate PR for each flow (cause that will be pretty big pr, check the issue https://github.com/status-im/status-desktop/issues/4920 )


> I think the problem was that once we switch to the real app, the animation kept playing even if it was no longer visible.
I guess the main problem is that we didn't remove the SplashScreen in that case, but still, we need to address the CPU issue and using playing: visible was a quick win.

@jrainville no, it won't be the case here, cause splash screen will be unloaded once we're ready to move to the app state, that means splash screen will be destroyed and won't be a chance for wasting any CPU time.